db/tests/functions/test_func_imports.py

```diff
@@ -0,0 +1,6 @@
+from db.functions.known_db_functions import known_db_functions
+
+
+def test_all_functions_no_raw_func():
```

Comment:
Could you add a docstring explaining why we're forbidding raw `func` calls?

Reply:
Sure, that's a good idea.